provider/cmd/pulumi-resource-cloud-toolkit-aws/kubernetes/adotApplication.ts

```diff
@@ -10,11 +10,13 @@ import { AdotApplicationArgs, AdotApplicationDefaultArgs } from "./adotApplicati
 export class AdotApplication extends ApplicationAddon<AdotApplicationArgs> {
   public readonly namespace?: kubernetes.core.v1.Namespace;
 
-  public readonly CWLogGroup: aws.cloudwatch.LogGroup;
+  public readonly logGroupApplicationLog?: aws.cloudwatch.LogGroup;
+  public readonly logGroupDataplaneLog?: aws.cloudwatch.LogGroup;
+  public readonly logGroupHostLog?: aws.cloudwatch.LogGroup;
+  public readonly logGroupMetrics?: aws.cloudwatch.LogGroup;
 
-  public readonly fluentBitIRSA: Irsa;
-
-  public readonly adotCollectorIRSA: Irsa;
+  public readonly fluentBitIRSA?: Irsa;
+  public readonly adotCollectorIRSA?: Irsa;
 
   public readonly application: kubernetes.apiextensions.CustomResource;
 
```

Comment:
Please, change class contructuctor to cloud-toolkit-aws:kubernetes:AdotApplication

Reply:
And the same in provider :)